packages/Webkul/DataTransfer/src/Queue/Worker.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Webkul\DataTransfer\Queue;
+
+use Illuminate\Queue\Worker as BaseWorker;
+use Illuminate\Queue\WorkerOptions;
+
+class Worker extends BaseWorker
+{
+    /**
+     * Listen to the given queue in a loop.
+     */
+    public function singleJobDaemon(string $connectionName, string $queue, WorkerOptions $options): void
+    {
+        if ($supportsAsyncSignals = $this->supportsAsyncSignals()) {
+            $this->listenForSignals();
+        }
+
+        while (true) {
+            if (isset($this->resetScope)) {
+                ($this->resetScope)();
+            }
+
+            // First, we will attempt to get the next job off of the queue. We will also
```

Comment:
Use multi line comments.

Reply:
I have to follow Laravel coding standard

```
https://github.com/laravel/framework/blob/11.x/src/Illuminate/Queue/Worker.php#L143
```

Reply:
Ok, since laravel framework files have comments the same way there is no need for this.